httpd-client/index.ts

```diff
@@ -115,6 +115,28 @@ export class HttpdClient {
     this.session = new session.Client(this.#fetcher);
   }
 
+  public changePort(port: number): void {
+    this.#baseUrl.port = port;
+  }
+
+  public get address(): string {
```

Comment:
Why not call this `url` as that is what it is?

Reply:
yeah we can do that 👍

Reply:
addressed in 039b625